docs/howtos/local_external_registry.md

```diff
@@ -0,0 +1,124 @@
+# Setup a local external registry
+
+## Certificates
+
+Before running the registry or the cluster we will need to create the certificates.
+
+We are going to put them in a `docker_reg_certs` folder:
+
+```
+mkdir docker_reg_certs
+```
+
+Using this `epinio.cfg` configuration file
+
+```
+[ req ]
+default_bits       = 2048
+distinguished_name = req_distinguished_name
+req_extensions     = req_ext
+[ req_distinguished_name ]
+countryName                = Country Name (2 letter code)
+stateOrProvinceName        = State or Province Name (full name)
+localityName               = Locality Name (eg, city)
+organizationName           = Organization Name (eg, company)
+commonName                 = Common Name (e.g. server FQDN or YOUR name)
+[ req_ext ]
+subjectAltName = @alt_names
+[alt_names]
+DNS.1   = omg.howdoi.website
+DNS.2   = *.omg.howdoi.website
+DNS.3   = 172.21.0.5.omg.howdoi.website
```

Comment:
I suspect that this entry has to be adapted to the local IP address, right ?
If yes, this should be noted IMHO.

Reply:
I should check if the wildcard is enough, or if the IP is needed. Maybe we can refine this to the smallest working configuration. I guess that it's full of all the possible combination because I was not sure on what fixed what.